proptest/Cargo.toml

```diff
@@ -56,6 +56,8 @@ atomic64bit = []
 # passed (see https://github.com/proptest-rs/proptest/issues/124).
 break-dead-code = []
 
+never-arbitrary = []
```

Comment:
Wasn't sure whether or not to just roll this in to the `unstable` feature flag. Opinions?

Reply:
Seems like a reasonable idea for stability as this is a fairly new area in the rust lang - (see my message below about coherence rules)